data/templates/load-balancing/haproxy.cfg.j2

```diff
@@ -0,0 +1,99 @@
+# Generated by /usr/libexec/vyos/conf_mode/load-balancing-haproxy.py
+
+global
+    log /dev/log local0
+    log /dev/log local1 notice
+    chroot /var/lib/haproxy
+    stats socket /run/haproxy/admin.sock mode 660 level admin
+    stats timeout 30s
+    user haproxy
+    group haproxy
+    daemon
+
+{% if global_parameters is vyos_defined %}
+{%     if global_parameters.max_connections is vyos_defined %}
+    maxconn {{ global_parameters.max_connections }}
+{%     endif %}
+
+    # Default SSL material locations
+    ca-base /etc/ssl/certs
+    crt-base /etc/ssl/private
+
+{%     if global_parameters.tls.ssl_bind_ciphers is vyos_defined %}
+    ssl-default-bind-ciphers {{ global_parameters.tls.ssl_bind_ciphers | join(':') | upper }}
+{%     else %}
+    # https://ssl-config.mozilla.org/#server=haproxy&version=2.6.12-1&config=intermediate&openssl=3.0.8-1&guideline=5.6
+    ssl-default-bind-ciphers ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-RSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:ECDHE-RSA-AES256-GCM-SHA384:ECDHE-ECDSA-CHACHA20-POLY1305:ECDHE-RSA-CHACHA20-POLY1305:DHE-RSA-AES128-GCM-SHA256:DHE-RSA-AES256-GCM-SHA384
```

Comment:
Please use `<defaultValue>` for defaults. Multiple defaults can be generated using whitespace separation.